include/boost/geometry/algorithms/detail/extreme_points.hpp

```diff
@@ -122,8 +122,8 @@ template<typename Ring, std::size_t Dimension>
 struct extreme_points_on_ring
 {
 
-    typedef typename geometry::coordinate_type<Ring>::type coordinate_type;
-    typedef typename geometry::point_type<Ring>::type point_type;
+    using coordinate_type = geometry::coordinate_type_t<Ring>;
+    using point_type = geometry::point_type_t<Ring>;
```

Comment:
The ones for `poitn_type_t` preceded by `geometry::` were missed in last PR and included now